### ISSUE TITLE
f20.sys missing

### ISSUE THREAD

**Issue by TheFausap** [g000001/Starlisp-simulator#1]
Hello,

the file f20.sys is missing here.
But you can find it in the zip file present at http://www.softwarepreservation.org/projects/LISP/parallel/#Connection_Machine_*Lisp_(StarLisp)_

regards,
Fausto

**Comment by g000001** [comment]
Hello, 

Thank you for your comment!
I removed f20.sys since I choose ASDF loading system, and to avoid confusing.
Does it make sense?

**Comment by TheFausap** [comment]
Yes, but I cannot find any reference to ASDF in the READMEs, and how to load that file using ASDF commands.
